samples/boards/mec15xxevb_assy6853/power_management/sample.yaml

```diff
@@ -2,5 +2,5 @@ sample:
   name: MEC150x board sample power management tests
 tests:
   sample.board.mec15xxevb_assy6853:
-    platform_allow: mec15xxevb_assy6853
+    skip: True
```

Comment:
Is there any difference with using `platform_exclude:` ?
Because in the documentation is written that usage of `skip` is "skip testcase unconditionally. This can be used for broken tests."
But for `platform_exclude:` is written that "Set of platforms that this test case should not run on."
https://docs.zephyrproject.org/latest/guides/test/sanitycheck.html?highlight=platform_exclude

I think more suitable will be usage of `platform_exclude`

Reply:
Yep, I have thought about this before, and as you see, the tag "platform_allow" means this test case just run at mec15xxevb_assy6853 board, if using "platform_exclude", I must ensure that all available boards are exclude, so maybe it's too long to write them I think. So @maksimmasalski , do you think this is OK? If so, I'll use platform_exclude tag instead. BTW, some impact will be met by using "skip" tag?

Reply:
Seems that app is written for the one platform.
I think when do that, you also have to update Readme and describe that sample is not suitable for sanitycheck, because you will receive timeout error, that is why it is skipped.

Reply:
You can also add into Readme description from that comment https://github.com/zephyrproject-rtos/zephyr/issues/28150#issuecomment-692294511

Reply:
Hi @maksimmasalski , I have updated the content in the README file.